app/src/main/java/com/practicum/playlistmaker/media/ui/activity/FavoriteFragment.kt

```diff
@@ -0,0 +1,32 @@
+package com.practicum.playlistmaker.media.ui.activity
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModel
+import com.practicum.playlistmaker.databinding.FragmentFavoriteBinding
+import com.practicum.playlistmaker.media.ui.view_model.FavoriteFragmentViewModel
+import com.practicum.playlistmaker.media.ui.view_model.PlaylistsFragmentViewModel
+import org.koin.androidx.viewmodel.ext.android.viewModel
+import org.koin.core.parameter.parametersOf
+
+class FavoriteFragment: Fragment() {
+
+    companion object{
+        fun newInstance()=FavoriteFragment()
+    }
+    private val viewModel by viewModel<FavoriteFragmentViewModel>()
+    private var _binding: FragmentFavoriteBinding?=null
+    private val  binding get() = _binding
```

Comment:
```suggestion
    private val  binding get() = _binding!!
```
Backing field делается для того, чтобы в коде фрагмента не приходилось каждый раз писать `!!`, поэтому этот оператор пишется один раз здесь)